token/cli/src/clap_app.rs

```diff
@@ -763,11 +763,34 @@ pub fn app<'a, 'b>(
                         .value_names(&["FEE_IN_BASIS_POINTS", "MAXIMUM_FEE"])
                         .takes_value(true)
                         .number_of_values(2)
+                        .hidden(true)
```

Comment:
Let's also make these conflict with the new ones, so that clap will raise the error for us
```suggestion
                        .hidden(true)
                        .conflicts_with("transfer_fee_basis_points")
                        .conflicts_with("transfer_fee_maximum_fee")
```